libs/sdk-ui-dashboard/src/model/events/layout.ts

```diff
@@ -597,6 +604,94 @@ export const isDashboardLayoutSectionItemMoved = eventGuard<DashboardLayoutSecti
 //
 //
 
+/**
+ * Payload of the {@link DashboardLayoutSectionItemMovedToNewSection} event.
+ * @alpha
+ */
+export interface DashboardLayoutSectionItemMovedToNewSectionPayload {
+    /**
+     * Item that was moved.
+     */
+    readonly item: ExtendedDashboardItem;
+
+    /**
+     * Index of section from which the item was moved.
+     */
+    readonly fromSectionIndex: number;
+
+    /**
+     * Index of section to which the item was moved.
+     *
+     * This may be the same as `fromSectionIndex` - which means the move happened within the same section.
+     */
+    readonly toSectionIndex: number;
+
+    /**
+     * Index within the `fromSection` from where the item was moved.
+     */
+    readonly fromIndex: number;
+
+    /**
+     * Index in `toSection` at which the item was inserted.
+     */
+    readonly toIndex: number;
```

Comment:
Is this always true? What if I move from section 0 to section 2 and section 0 is removed? Then this would say 2, but the item would actually be in section1, no?

Reply:
I have updated the command handler, index is recalculated after the previous section removed.